### ISSUE TITLE
[IDEA] unlock (Intel) GPU features by checking

### ISSUE THREAD

**Issue by forart** [cdgriffith/FastFlix#389]
It seems that @rigaya is working hard on QSVEnc's "--check-features" function and it would be cool to use for en/dis-able (but not force) relative accelerations in FF.

This may help users to understand better what they can or cannot do with their hardware and prevent possible misusing.

Hope that inspires.

**Comment by cdgriffith** [comment]
Yeah I really want to find a way I can easily just read what the user's hardware is without downloading all three encoders and doing a check on each so I can first decided which to even suggest downloading. But you're right that only supporting each actually supported subset is also important!

**Comment by forart** [comment]
OK, just checked: in @StaxRip they filter available-on-your-hw features under "Encoder Options" before opening it.

As you can see, for example, here's available hw AVC features detected by QSVEnc...
![AVC_features](https://user-images.githubusercontent.com/171307/229700824-d0a6225e-104c-4a24-b4da-601123c63f2f.png)

...and the StaxRip menu:
![AVC_staxrip](https://user-images.githubusercontent.com/171307/229701246-9987c30b-f003-4324-b579-53669fd68095.png)

Countercheck (MPEG-2):
![MPEG2_features](https://user-images.githubusercontent.com/171307/229701659-1387d1b1-fb35-47b8-9769-8272b9acf06f.png)
![MPEG2_staxrip](https://user-images.githubusercontent.com/171307/229701959-2bf214d0-e65c-44d4-863a-53ee8bb61918.png)

Hope that inspires !

**Comment by cdgriffith** [comment]
Adding detection of at least supported encoders in https://github.com/cdgriffith/FastFlix/releases/tag/5.4.0 but agreed would be good to go that extra step and parse features for each as well!
